docs/paper-joss/paper.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'MRI-NUFFT: Doing non-Cartesian MRI has never been easier'
+tags:
+  - Python
+  - MRI
+  - NUFFT
+  - Numpy 
+  - CUDA
+  - Torch 
+authors:
+  - name: Pierre-Antoine Comby
+    orcid: 0000-0001-6998-232X
+    corresponding: true
+    affiliation: "1, 2"
+  - name: Guillaume Daval-Frérot
+    affiliation: 3
+  - name: Asma Tanaben
+    affiliation: "1,2,5" 
+  - name: Caini Pan 
+    affiliation: "1, 2"
+  - name: Léna Oudjman
+    affiliation: 1,2
+  - name: Matteo Cencini
+    affiliation: 4
+  - name: Philippe Ciuciu
+    affiliation: "1,2"
+  - name: Chaithya GR
+    corresponding: true 
+    affiliation: "1,2"
+    
+affiliations:
+ - name: MIND, Inria
+   index: 1
+ - name: Université Paris-Saclay / CEA 
+   index: 2
+ - name: Chipiron
+   index: 3
+ - name: INFN, Pisa Division 
+   index: 4 
+ - name: Siemens Healthineers 
+   index: 5
+   
+date: 20 September 2024
+bibliography: paper.bib
+---
+
+
+# Summary 
+MRI-NUFFT is a python package that provides a universal interface to various Non-Uniform Fast Fourier Transforms libraries running on CPU or GPU (gpunufft, finufft, cufinufft, pynfft), adding compatibily with standard array library (numpy, cupy, torch, tensorflow, etc.) On top of these libraries it extends the existing NUFFT operations to provide a physical model of the MRI acquisition process (e.g. multi-coil acquisition and static-field inhomogeneities). Moreover, it also provides a wide variety of non-Cartesian Sampling trajectories generation and expansion, as well as density-compensation estimation methods for those trajectories. It also implements optimized auto-differentiation with respect to the data and the sampling locations. With MRI-NUFFT one can experiment with non-cartesian sampling in MRI, get access to the latest advances in the field and state-of-the art sampling patterns.
```

Comment:
I am still not sure because of the logo and the varying names, but is it MRI-_nufft_, or MRI-NUFFT ?

```suggestion
MRI-NUFFT is a python package that provides a universal interface to various Non-Uniform Fast Fourier Transform libraries running on CPU or GPU (gpuNUFFT, FINUFFT, CUFINUFFT, pyNFFT), adding compatibily with standard array library (NumPy, CuPy, PyTorch, TensorFlow, etc.) On top of these libraries it extends the existing NUFFT operations to provide a physical model of the MRI acquisition process (e.g. multi-coil acquisition and static-field inhomogeneities). It also provides a wide variety of customizable implementations of non-Cartesian sampling trajectories, as well as density compensation methods. Finally, it proposes optimized auto-differentiation with respect to the data and sampling locations for machine learning. With MRI-NUFFT one can experiment with non-Cartesian sampling in MRI, get access to the latest advances in the field and state-of-the-art sampling patterns.
```

Reply:
I have settle with MRI-NUFFT in all caps everywhere, except in the logo...